RecoPixelVertexing/PixelLowPtUtilities/src/StripSubClusterShapeTrajectoryFilter.cc

```diff
@@ -196,6 +194,25 @@ StripSubClusterShapeFilterBase::~StripSubClusterShapeFilterBase() {
 #endif
 }
 
+void StripSubClusterShapeFilterBase::fillPSetDescription(edm::ParameterSetDescription &iDesc) {
+  iDesc.addUntracked<std::string>("label", "");
+  iDesc.add<uint32_t>("maxNSat", 3);
+  iDesc.add<double>("trimMaxADC", 30.);
+  iDesc.add<double>("trimMaxFracTotal", .15);
+  iDesc.add<double>("trimMaxFracNeigh", .25);
+  iDesc.add<double>("maxTrimmedSizeDiffPos", .7);
+  iDesc.add<double>("maxTrimmedSizeDiffNeg", 1.);
+  iDesc.add<double>("subclusterWindow", .7);
+  iDesc.add<double>("seedCutMIPs", .35);
+  iDesc.add<double>("seedCutSN", 7.);
+  iDesc.add<double>("subclusterCutMIPs", .45);
+  iDesc.add<double>("subclusterCutSN", 12.);
+
+  edm::ParameterSetDescription psdLM;
+  psdLM.setAllowAnything();
```

Comment:
> psdLM.setAllowAnything();

Just to note that, here, there is still a shortcut. The treatment of this PSet is [not entirely trivial](https://github.com/cms-sw/cmssw/blob/21d4e797879416bfedc949c62edc3feb89489ddd/RecoPixelVertexing/PixelLowPtUtilities/src/StripSubClusterShapeTrajectoryFilter.cc#L163-L183), so further improvements are left to experts.